### DISCUSSION TITLE
v2.0 中 createForm 的使用方式，是否需要用useMemo包一下

### DISCUSSION THREAD

**Discussion by Nokecy** [alibaba/formily#1223] [Q&A 1.x] [answered]
`         const form = useMemo(() => createForm({ }), []);  `

**Comment by janryWang** [comment, marked as answer]
目前createForm是支持单例模式和多实例模式的，这个主要根据用户场景来看，比如用户场景是SPA应用，那必然是走多实例模式要更好，因为方便内存回收，当然单例模式是会在FormProvider被卸载的时候回收Form实例内部的所有监听器，只是闭包内不能回收Form实例这个引用
